src/screens/Settings/index.tsx

```diff
@@ -250,7 +263,9 @@ function Settings() {
     audioFix,
     autoInstallDxvk,
     autoSyncSaves,
+    enableEsync,
```

Comment:
I guess we also need those props on the element above otherwise the general wine settings might not get updated.

Reply:
Got them in there.